addons/resource/views/resource_calendar_attendance_views.xml

```diff
@@ -16,7 +16,7 @@
                 <field name="duration_days" optional="show"/>
                 <field name="date_from" optional="hide"/>
                 <field name="date_to" optional="hide"/>
-                <field name="week_type" readonly="1" force_save="1" groups="base.group_no_one"/>
+                <field name="week_type" readonly="1" force_save="1" optional="hide"/>
```

Comment:
alternatively if we wanted to keep the group_no_one, I think we often do this in form views that probably work in list view too:

```suggestion
                <field name="week_type" readonly="1" force_save="1" groups="base.group_no_one"/>
                <field name="week_type" invisible="1" groups="!base.group_no_one"/>
```
but if it's necessary for the order, it seems to make sense to have it possibly shown to non-group no one users.